about.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Lato" />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/d334bd1716.js" crossorigin="anonymous"></script>
+  <!-- Bootstrap CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/about.css">
+    <title>About Kadoma Community Development</title>
+</head>
+<body>
+    <nav>
+        <ul class="d-lg-flex d-none justify-content-end sp-bg list-unstyled">
+            <li class="px-1 py-1">
+                <a href="#"><i class="fa fa-facebook"></i> </a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">Logout</a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">My Page </a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">English</a>
+            </li>
+        </ul>
+    </nav>
+    <nav class="navbar navbar-expand-lg">
```

Comment:
* Kindly fix this navbar section as it is diiferent from the original design and the page overflows to the right HINT 💭 : you can change the size of the logo and text so that the nav links would fit the screen size 👍 
![image](https://user-images.githubusercontent.com/52497246/133552986-325ff176-005e-4045-8d68-6725e22417ec.png)